tests/UnifiedSpecTests/transactions/retryable-abort-handshake.json

```diff
@@ -0,0 +1,204 @@
+{
+  "description": "retryable abortTransaction on handshake errors",
+  "schemaVersion": "1.4",
+  "runOnRequirements": [
+    {
+      "minServerVersion": "4.2",
+      "topologies": [
+        "replicaset",
+        "sharded",
+        "load-balanced"
+      ],
+      "serverless": "forbid",
+      "auth": true
+    }
+  ],
+  "createEntities": [
+    {
+      "client": {
+        "id": "client0",
+        "useMultipleMongoses": false,
+        "observeEvents": [
+          "commandStartedEvent",
+          "connectionCheckOutStartedEvent"
+        ]
+      }
+    },
+    {
+      "database": {
+        "id": "database0",
+        "client": "client0",
+        "databaseName": "retryable-handshake-tests"
+      }
+    },
+    {
+      "collection": {
+        "id": "collection0",
+        "database": "database0",
+        "collectionName": "coll"
+      }
+    },
+    {
+      "session": {
+        "id": "session0",
+        "client": "client0"
+      }
+    },
+    {
+      "session": {
+        "id": "session1",
+        "client": "client0"
+      }
+    }
+  ],
+  "initialData": [
+    {
+      "collectionName": "coll",
+      "databaseName": "retryable-handshake-tests",
+      "documents": [
+        {
+          "_id": 1,
+          "x": 11
+        }
+      ]
+    }
+  ],
+  "tests": [
+    {
+      "description": "AbortTransaction succeeds after handshake network error",
+      "skipReason": "DRIVERS-2032: Pinned servers need to be checked if they are still selectable",
```

Comment:
I noted from the downstream changes in [DRIVERS-746](https://jira.mongodb.org/browse/DRIVERS-746) that both of the "retryable" tests are skipped, so it was not necessary to add these to `UnifiedSpecTest::$incompleteTests`.

Reply:
I noted it too and that is why I didn't add new lines in UnifiedSpecTest::$incompleteTests. Also I double checked the [DRIVERS-2032](https://jira.mongodb.org/browse/DRIVERS-2032) ticket which is the parent ticket of [DRIVERS-746](https://jira.mongodb.org/browse/DRIVERS-746).